src/sdk/CMakeLists.txt

```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if(TESTING_ENABLE)
+# coverage test won't run tests below, but they are *_test, so if COVERAGE_NO_DEPS, disable build
+if(TESTING_ENABLE AND NOT COVERAGE_NO_DEPS)
```

Comment:
it do not update the `test_list` variable, why coverage pull the targets in sdk ?

This commit introduce : https://github.com/4paradigm/OpenMLDB/blob/10721cd05540ba51f3c0eb309bca819cf9832eb8/src/CMakeLists.txt#L98 ?

Reply:
I split test build in workflow:
`make first 60 tests, clean .o/... files, make other tests`
 just grep the target name end with _test, so the tests here is including, and hard to filter, so disable in cmake.

Reply:
leave it